apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.js

```diff
@@ -63,7 +62,6 @@ function HelpCenterContent() {
 				<>
 					<PinnedItems scope="core/edit-post">{ content }</PinnedItems>
 					<PinnedItems scope="core/edit-site">{ content }</PinnedItems>
-					<PinnedItems scope="core/edit-widgets">{ content }</PinnedItems>
```

Comment:
This seems incorrect. The widgets page lost the Help Center in both headers. 
<img width="625" alt="image" src="https://user-images.githubusercontent.com/17054134/191940726-18c2c871-41d4-4ff2-9c59-a396ae456ec5.png">